monitor_check.py

```diff
@@ -279,6 +335,52 @@ def bug(bugs, package):
 
 counter = Counter()
 
+def pkgname(srcname):
+    pkgs = repoquery(source=srcname.rsplit("-", 2)[0], version="rawhide")
+    for pkg in pkgs:  # a only gets evaluated here
+    #    if pkg.reponame == "fedorarawhide":
+        return pkg.source_name
+
+def rawhide_sack():
+    """A DNF sack for rawhide, used for queries, cached"""
+    try:
+        return sacks[RAWHIDEVER]
+    except KeyError:
+        pass
+    base = dnf.Base()
+    conf = base.conf
+    conf.cachedir = DNF_CACHEDIR
+    conf.substitutions['releasever'] = str(RAWHIDEVER)
+    conf.substitutions['basearch'] = ARCH
+    base.repos.add_new_repo('rawhide', conf,
+        metalink='https://mirrors.fedoraproject.org/metalink?repo=rawhide&arch=$basearch',
```

Comment:
You might want to use the koji repo instead for more up to date data. It doe snot have `metalink` but `baseurl` and that needs to be a list :shrug: 

```suggestion
        baseurl=['http://kojipkgs.fedoraproject.org/repos/rawhide/latest/$basearch/'],
```